src/splat/segtypes/n64/asm.py

```diff
@@ -22,7 +21,12 @@ def get_file_header():
         if preamble:
             ret.append(preamble)
             ret.append("")
-        ret.append('.section .text, "ax"')
+
+        line = f".section {self.get_linker_section_linksection()}"
+        section_flags = self.get_section_flags()
+        if section_flags:
+            line += f', "{section_flags}"'
+        ret.append(line)
```

Comment:
Could this stuff be in a function in common/asm and we just call that from these subclass implementations? cause it seems to be the same thing repeated 4 times in all the subclasses

Reply:
dun